pkg/internal/venafi/sign.go

```diff
@@ -152,6 +152,12 @@ func (v *Venafi) Sign(csrPEM []byte, duration time.Duration, customFields []inte
 
 func newVRequest(cert *x509.Certificate) *certificate.Request {
 	req := certificate.NewRequest(cert)
+
+	if cert.Subject.String() == "" {
```

Comment:
Do we want to check if `cert.Subject.String()` is empty? It seems like we'd be better to check if specifically the organization is empty, and if so, set it? Else we have a weird behaviour where if a user specifies a `postCode` or something, they don't get a default organization.. which feels weird 😅

Reply:
Sure!